bodhi/client/__init__.py

```diff
@@ -84,6 +84,39 @@ def new(user, password, **kwargs):
 
 
 @updates.command()
+@click.argument('builds')
+@click.option('--user', envvar='USERNAME')
+@click.option('--password', prompt=True, hide_input=True)
+@click.option('--type', default='bugfix', help='Update type', required=True,
+              type=click.Choice(['security', 'bugfix',
+                                 'enhancement', 'newpackage']))
+@click.option('--notes', help='Update description')
+@click.option('--notes-file', help='Update description from a file')
+@click.option('--bugs', help='Comma-seperated list of bug numbers', default='')
+@click.option('--close-bugs', default=True, is_flag=True, help='Automatically close bugs')
+@click.option('--request', help='Requested repository',
+              type=click.Choice(['testing', 'stable', 'unpush']))
+@click.option('--autokarma', default=True, is_flag=True, help='Enable karma automatism')
+@click.option('--stable-karma', type=click.INT, help='Stable karma threshold')
+@click.option('--unstable-karma', type=click.INT, help='Unstable karma threshold')
+@click.option('--suggest', help='Post-update user suggestion',
+              type=click.Choice(['logout', 'reboot']))
+@click.option('--file', help='A text file containing all the update details')
+@click.option('--staging', help='Use the staging bodhi instance',
+              is_flag=True, default=False)
+def edit(user, password, **kwargs):
+    client = bindings.BodhiClient(username=user, password=password, staging=kwargs['staging'])
+    kwargs['edited'] = kwargs['builds']
+    try:
+        resp = client.save(**kwargs)
+        print_resp(resp, client)
+    except bindings.BodhiClientException as e:
+        click.echo(str(e))
+    except Exception as e:
+        traceback.print_exc()
```

Comment:
I realize that the new() function does this too, but if we're going to print a traceback and exit, you might as well just not catch this Exception and let Python crash (i.e., I think you can delete the above two lines).